### DISCUSSION TITLE
Feature request: Additional fields for data flows

### DISCUSSION THREAD

**Discussion by misch42** [Alef-Burzmali/netbox-data-flows#53] [Ideas]
I'd like to see some additional fields on the data flows. That would be bandwith and latency. What would be the average bandwidth for a data flow and what would be the allowed latency for a data flow?

At the moment I do this with custom fields, but fields in the model would be a good idea.

**Comment by Alef-Burzmali** [comment]
Hello
I don't think that these fields would be relevant for the majority of the plugin's users (and I would not use them either). As you said, you can already create custom fields for your specific needs.

However, I'm opened to be convinced otherwise. If other people feel these fields would be useful, please chime in and explain what's your use case.
